Examples/Python/ellipsoid.py

```diff
@@ -240,6 +240,20 @@ def Run_Pipeline(args):
         dt_files = sw.utils.save_images(groom_dir + 'distance_transforms/', shape_seg_list,
                                         shape_names, extension='nrrd', compressed=False, verbose=True)
 
+    # If running optimzation in mesh mode, convert distance transforms to meshes
+    if args.mesh_mode:
+        # Make output folder
+        mesh_dir = groom_dir + 'meshes/'
+        if not os.path.exists(mesh_dir):
+            os.mkdir(mesh_dir)
+        # Convert distance transforms to meshes
+        mesh_files = []
+        for dt, shape_name in zip(shape_seg_list, shape_names):
+            print('Ceating mesh from: ' + shape_name)
+            dt.toMesh(0).write(mesh_dir + shape_name + ".ply")
+            mesh_files.append(mesh_dir + shape_name + ".ply")
```

Comment:
This gives a warning:
```WARNING: Trying to save mesh with new field. Only vtk and vtp files save associated fields
```

Reply:
I am not seeing that warning but I will switch it to vtk